.github/workflows/build-with-ballerina-lang.yml

```diff
@@ -0,0 +1,122 @@
+name: Build with Ballerina Lang
```

Comment:
Do we need this workflow? Does the other connector modules have this workflow?

Reply:
I got these workflows from the `ballerinax/kafka` connector, since these have the compiler plugin I was asked to add the standard library workflows. https://github.com/ballerina-platform/module-ballerinax-kafka/blob/master/.github/workflows/build-with-ballerina-lang.yml